src/styles.tsx

```diff
@@ -0,0 +1,43 @@
+export const color={
+    primary:"#00b4d8",
+    secondary:"#00b4d8",
+    danger:"#00b4d8",
+    success:"#00b4d8",
+    warning:"#00b4d8",
+};
+export const boxHW = {
+      height:{
+        small:"10rem",
+        medium:"30rem",
+        large:"50rem"
+      },
+      width:{
+        small:"10rem",
+        medium:"30rem",
+        large:"50rem"
+      }
+};
```

Comment:
Box height and width should not be set directly. Allow the components to take the natural space. Managing responsiveness is easier when size is not set

Reply:
This was a demo component, and to try the storybook things I made the height and width static.

Reply:
But for our main components, we will make it responsive.